arm-template-retrieve-azure-storage-access-keys/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "storageAccountName": {
+        "value": "GEN-UNIQUE"
+      },
+      "storageAccountSku": {
```

Comment:
remove the parameters you have defaultValues for... so deployment can test your defaultValues (Location you have is fine if you want to deploy there)